[1.18.2] Add Traditional Chinese Language

Jimmy-sheep opened the pull request:
Add Traditional Chinese Language, copy from zh_cn.json, but change about 10% of phraseology.

Commit:
Add zh_tw.json

Base on zh_cn, but change some phraseology.

Commit:
Update zh_tw.json

Change some phraseology and proofread.

Commit:
Update zh_tw.json

Change one phraseology.

StarskyXIII commented:
一堆未在地化的內容，請確實與原版翻譯對照後再進行 PR。
10% 的用語是指繁化姬轉換的嗎？
連遊戲都沒打開測試過？

Jimmy-sheep commented:
> 一堆未在地化的內容，請確實與原版翻譯對照後再進行 PR。 10% 的用語是指繁化姬轉換的嗎？ 連遊戲都沒打開測試過？

這是給1.18.2版本使用的 對照en_us語言 我想應該是沒有遺漏很多 用語部份是我自己純手動用Notepad++更改的並不是用機翻 (我在此之前也沒聽過繁化姬) 也許有些部分會遺漏 也有些兩岸用語我搞混而沒翻到 可以提出來我會再修正
這也因為和朋友玩後看到1.18.2版本沒有繁體中文而第一次嘗試自己翻譯 所以有些格式不一定能夠符合要求

Commit:
Update zh_tw.json

According to the suggestion from StarskyXIII, fix some phraseology.

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.oil_sand": "瀝青沙",
+  "block.thermal.oil_red_sand": "瀝青紅沙",
+
+  "block.thermal.apatite_block": "磷灰石塊",
```
該類型方塊，原版幾乎都是譯作為 `OO方塊`，看你要不要參考看看

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.steel_block": "鋼塊",
+  "block.thermal.rose_gold_block": "玫瑰金塊",
+
+  "block.thermal.enderium_block": "終界塊",
```
這邊只是建議，由於有多款模組分別加入了 `End` `Ender` 之類的金屬，
故為了做出區別，這邊可以譯作 `終界合金`

StarskyXIII replied:
我個人對舊版本接觸較少，但根據我查閱的對岸資料（早期台灣翻譯幾乎都為直接簡繁轉換而來），
https://www.mcmod.cn/item/9425.html
該金屬原先就稱作為 `Enderium Ingot`，也就是 `終界錠`。
諧振終界 `Resonant Ender` 為流體專屬的特殊名稱，
類似特殊的命名還有不穩定紅石 `Destabilized Redstone`（將紅石送入熔岩爐熔化）。
故我對將錠與流體統一命名為 `諧振` 持保留的態度。

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.signalum_glass": "信素玻璃",
+
+  "block.thermal.white_rockwool": "白色石棉",
+  "block.thermal.orange_rockwool": "橘色石棉",
```
Orange 原版譯作 `橙色`

Jimmy-sheep replied:
抱歉這是我的失誤 我誤記成原版是橘色

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.red_rockwool": "紅色石棉",
+  "block.thermal.black_rockwool": "黑色石棉",
+
+  "block.thermal.chiseled_rich_slag": "鑿製富爐渣塊",
```
Chiseled 原版譯作 `浮雕`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.chiseled_slag": "鑿製爐渣塊",
+  "block.thermal.cracked_rich_slag_bricks": "裂紋富爐渣塊",
+  "block.thermal.cracked_slag_bricks": "裂紋爐渣塊",
+  "block.thermal.polished_rich_slag": "燒製富爐渣塊",
```
Polished 原版譯作 `拋光`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.slag_block": "爐渣塊",
+  "block.thermal.tar_block": "焦油塊",
+
+  "death.attack.basalz": "%1$s的世界天翻地覆！",
```
變數為玩家名時，建議與中文字間留空格

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "death.attack.blitz": "%1$s得到了悲慘的教訓！",
+  "death.attack.blizz": "%1$s邂逅了冷酷的命運！",
+
+  "entity.thermal.basalz": "岩石人",
```
Blaze 原版譯作 `烈焰使者`，這邊 `人` 改譯作 `使者` 是否比較妥當呢？

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "info.thermal.detonator.use.1": "使用以引爆綁定的炸藥。",
+  "info.thermal.detonator.use.sneak": "潛行並使用以清空綁定。",
+
+  "info.thermal.lock.use": "對可兼容方塊使用來為其上鎖。",
```
相容

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "item.thermal.resin_bucket": "樹脂桶",
+  "item.thermal.sap_bucket": "樹汁桶",
+  "item.thermal.seed_oil_bucket": "種子油桶",
+  "item.thermal.syrup_bottle": "瓶裝糖漿",
```
與蜂蜜瓶類似，翻成 `糖漿瓶` 應該比較好

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+
+  "item.thermal.apatite": "磷灰石",
+  "item.thermal.apatite_dust": "磷灰石粉",
+  "item.thermal.cinnabar": "朱砂",
```
硃砂

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+
+  "item.thermal.basalz_powder": "岩石粉",
+  "item.thermal.basalz_rod": "岩石裂骸",
+  "item.thermal.basalz_spawn_egg": "岩石人刷怪蛋",
```
參考格式：`岩石使者 生怪蛋`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "item.thermal.junk_net": "垃圾打撈網",
+  "item.thermal.junk_net.desc": "用於在§e§o水產捕撈器§r§6中打撈垃圾。不會被消耗。",
+
+  "item.thermal.raw_lead": "粗鉛",
```
原版譯作 `O原礦`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "item.thermal.electrum_nugget": "琥珀金粒",
+  "item.thermal.electrum_plate": "琥珀金板",
+
+  "item.thermal.invar_coin": "殷鋼幣",
```
台灣這邊沒 `殷鋼` 的用法，建議改譯作 `不變鋼`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+
+  "subtitles.thermal.block.device_water_gen": "蓄水器冒泡",
+
+  "subtitles.thermal.entity.basalz.ambient": "岩石人呼吸",
```
Blaze 原版譯作 `烈焰使者`，這邊 `人` 改譯作 `使者` 是否比較妥當呢？

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "item.thermal.steel_nugget": "鋼粒",
+  "item.thermal.steel_plate": "鋼板",
+
+  "item.thermal.enderium_coin": "終界幣",
```
這邊只是建議，由於有多款模組分別加入了 `End` `Ender` 之類的金屬，
故為了做出區別，這邊可以譯作 `終界合金`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "info.thermal.augment.attr.PotionDur": "效果時長",
+
+  "info.thermal.augment.attr.RFMax": "RF容量",
+  "info.thermal.augment.attr.RFXfer": "RF傳輸",
```
英文與中文間建議留空格

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+
+  "item.thermal.fluid_tank_augment": "擴容儲罐構造",
+
+  "item.thermal.fluid_tank_creative_augment": "創造儲罐建構",
```
Creative 建議譯作 `創造模式`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "item.thermal.machine_output_augment": "輔助處理篩",
+  "item.thermal.machine_output_augment.desc": "增加副產物的產生",
+  "item.thermal.machine_speed_augment": "通量鏈接放大器",
+  "item.thermal.machine_speed_augment.desc": "以RF效率降低為代價提升處理速度。",
```
英文與中文間建議留空格

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "item.thermal.rs_control_augment": "紅石反應機制",
+  "item.thermal.rs_control_augment.desc": "允許紅石訊號控制。",
+
+  "item.thermal.side_config_augment": "模塊設置重構",
```
`Module` 台灣這邊說法是 `模組`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "item.thermal.diving_boots": "潛水腳蹼",
+
+  "item.thermal.hazmat_fabric": "毒害隔離織物",
+  "item.thermal.hazmat_helmet": "防護盔",
```
個人感覺盔甲照原版格式翻譯會比較合適，也更方便玩家在 JEI 搜尋相同類型東西
`頭盔` `胸甲` `護腿` `靴子`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+
+  "block.thermal.bamboo_block": "竹捆",
+  "block.thermal.charcoal_block": "木炭塊",
+  "block.thermal.gunpowder_block": "壓實火藥塊",
```
不太理解為何會冒出 `壓實`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "_comment": "========================================================================================================",
+
+  "block.thermal.bamboo_block": "竹捆",
+  "block.thermal.charcoal_block": "木炭塊",
```
該類型方塊，原版幾乎都是譯作為 `OO方塊`，看你要不要參考看看

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.carrot_block": "箱裝胡蘿蔔",
+  "block.thermal.potato_block": "箱裝馬鈴薯",
+
+  "block.thermal.raw_lead_block": "粗鉛塊",
```
原版譯作為 `O原礦方塊`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "_comment": "Keywords - These terms help things show up in JEI or other tooltip searches! :)",
+  "_comment": "========================================================================================================",
+
+  "block.thermal.earth_tnt.keyword": "岩石人 地 tnt",
```
Blaze 原版譯作 `烈焰使者`，這邊 `人` 改譯作 `使者` 是否比較妥當呢？

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.charge_bench.keyword": "能量 充電器 rf",
+  "block.thermal.tinker_bench.keyword": "組件 充電器 灌注 裝填 注水 升級",
+
+  "block.thermal.chunk_loader.keyword": "區塊 區塊加載器 加載器",
```
載入

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.tomato": "番茄",
+  "block.thermal.tomato_block": "箱裝番茄",
+
+  "death.attack.sadiroot": "%1$s被薩蒂根莖的刺葉扎死了",
```
變數為玩家名時，建議與中文字間留空格

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "item.thermal.tomato": "番茄",
+  "item.thermal.tomato_seeds": "番茄種子",
+
+  "item.thermal.dough": "面團",
```
麵團...

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.energy_duct": "能量導管",
+  "block.thermal.fluid_duct": "流體導管",
+
+  "block.thermal.energy_distributor": "RF分發接口",
```
接口，台灣這邊說法是 `介面`
雖然這模組還沒有發布，但這翻譯真的很奇怪，看你要不要修改一下

Jimmy-sheep replied:
> 接口，台灣這邊說法是 `介面` 雖然這模組還沒有發布，但這翻譯真的很奇怪，看你要不要修改一下

雖然英文版適用interface但...這部分我在想...也許翻成`埠`更合適?

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+
+  "info.thermal.item_buffer.0": "限制物品的傳送速率和數量。",
+  "info.thermal.item_buffer.1": "使用左側網格來設置所需物品及其數量",
+  "info.thermal.item_buffer.2": "鎖存模式下，只有所有物品均滿足要求才會輸出。",
```
鎖存，台灣這邊說法是 `閂鎖`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "item.thermal.press_packing_3x3_die": "打包模具（3x3）",
+  "item.thermal.press_unpacking_die": "解包模具",
+
+  "item.thermal.slot_seal": "槽楔",
```
~~感覺有點太文青了~~，我無法想像玩家交談時，「槽楔」「槽楔」的唸

Jimmy-sheep replied:
> ~感覺有點太文青了~，我無法想像玩家交談時，「槽楔」「槽楔」的唸

確實 但這物品確實更接近"楔子"這種東西...
又或者是翻譯成`欄位卡榫`呢...?

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "item.thermal.press_unpacking_die": "解包模具",
+
+  "item.thermal.slot_seal": "槽楔",
+  "item.thermal.slot_seal.desc": "防止所在槽位被使用。適用於有多個輸入槽的機器。",
```
原版 Slot 會譯作為 `欄位`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+
+  "block.thermal.crossover_rail": "渡線鐵軌",
+
+  "block.thermal.lumium_activator_rail": "流明激活鐵軌",
```
`Activator Rail` 原版翻 `觸發鐵軌`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.lumium_powered_rail": "流明充能鐵軌",
+  "block.thermal.lumium_rail": "流明鐵軌",
+
+  "block.thermal.prismarine_activator_rail": "海晶激活鐵軌",
```
原版譯作 `海磷`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "entity.thermal.ice_tnt_minecart": "冰川結凍炸藥礦車",
+  "entity.thermal.lightning_tnt_minecart": "震天雷擊炸藥礦車",
+  "entity.thermal.nuke_tnt_minecart": "核彈礦車",
+  "entity.thermal.phyto_tnt_minecart": "PNT礦車",
```
英文建議與中文間留空

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.prismarine_powered_rail.keyword": "水下",
+  "block.thermal.prismarine_rail.keyword": "水下",
+
+  "item.thermal.earth_tnt_minecart.keyword": "岩石人 地 tnt",
```
Blaze 原版譯作 `烈焰使者`，這邊 `人` 改譯作 `使者` 是否比較妥當呢？

Commit:
Update zh_tw.json

According to the suggestion from StarskyXIII, fix some phraseology.

Here is a translation, still need comfirm:
    "item.thermal.slot_seal": "槽楔"

Thanks for StarskyXIII support the translations.

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.rich_slag_block": "富爐渣方塊",
+  "block.thermal.rosin_block": "松香方塊",
+  "block.thermal.rubber_block": "橡膠方塊",
+  "block.thermal.sawdust_block": "壓實鋸屑方塊",
```
壓實感覺是多餘的

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "info.thermal.detonator.use.1": "使用以引爆綁定的炸藥。",
+  "info.thermal.detonator.use.sneak": "潛行並使用以清空綁定。",
+
+  "info.thermal.lock.use": "對可相容方塊使用為其上鎖。",
```
`可` 感覺是贅字

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "info.thermal.satchel.mode.0": "自動儲存：關閉",
+  "info.thermal.satchel.mode.1": "自動儲存：開啟",
+  "info.thermal.satchel.use": "使用以打開背包。",
+  "info.thermal.satchel.use.sneak": "潛行並使用以打開物品篩選界面。",
```
台灣這邊應用 `介面`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "item.thermal.blizz_spawn_egg": "暴雪使者 生怪蛋",
+
+  "item.thermal.compost": "堆肥",
+  "item.thermal.detonator": "遙控起爆器",
```
引爆器是否會更合適

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "item.thermal.machine_null_augment.desc": "銷毀多餘的副產物。",
+  "item.thermal.machine_output_augment": "輔助處理篩",
+  "item.thermal.machine_output_augment.desc": "增加副產物的產生",
+  "item.thermal.machine_speed_augment": "通量鏈接放大器",
```
台灣這邊應用 `連結`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.ice_tnt": "冰川結凍炸藥",
+  "block.thermal.ice_tnt.desc": "凍結一大片區域。",
+  "block.thermal.lightning_tnt": "震天雷擊炸藥",
+  "block.thermal.lightning_tnt.desc": "震擊一大片區域，起效時有很大機率伴有雷電。",
```
`起作用` 或 `生效` 會不會比較合適

Jimmy-sheep replied:
我剛對照了英文版的這部分 -> `Very likely to bring the thunder.`
我想..省略生效部分應該比較符合原文...? -> `很高機率伴有雷電。`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.nuke_tnt.desc": "千萬，千萬別站得太近。",
+  "block.thermal.phyto_tnt": "PNT",
+  "block.thermal.phyto_tnt.desc": "大範圍噴灑植育一號，非投擲物。",
+  "block.thermal.redstone_tnt": "紅石發信炸藥",
```
這邊我不太肯定，但台灣這邊應該是使用 `發射` 或 `訊號發射`

Jimmy-sheep replied:
這邊我改翻譯為 `紅石訊號炸藥` 這個方塊似乎是爆炸時會大範圍產生紅石訊號 (而且翻譯成6個字與其他tnt比較統一)

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+
+  "info.thermal.device_nullifier.0": "銷毀任何被放入的東西",
+  "info.thermal.device_nullifier.1": "自動輸入的物品與流體將被立即銷毀。",
+  "info.thermal.device_nullifier.2": "手動輸入的物品必須通過點擊按鈕進行銷毀。",
```
通過是指你行經一個地方，應用 `透過`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "info.thermal.machine_brewer.1": "你可能會想要擺上一整排。",
+  "info.thermal.machine_brewer.2": "不需要玻璃瓶。",
+
+  "info.thermal.machine_centrifuge.0": "通過高速旋轉離心技術將物品分離成其構成材料。",
```
通過是指你行經一個地方，應用 `透過`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "item.thermal.beekeeper_leggings": "養蜂人長褲",
+  "item.thermal.beekeeper_boots": "養蜂人靴子",
+
+  "item.thermal.diving_fabric": "海磷面料",
```
應用 `布料`

Jimmy-sheep replied:
這邊我改翻譯為`防水布料` 似乎比較貼近原文 也符合用途

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "item.thermal.diving_leggings": "潛水護腿",
+  "item.thermal.diving_boots": "潛水腳蹼",
+
+  "item.thermal.hazmat_fabric": "毒害隔離織物",
```
形狀與 `Marine Fabric` 相同，同用 `布料` 就好

Jimmy-sheep replied:
原文為`Protective Fabric` 但考慮到這個物品製作的套裝 有防護各種異常效果(中毒 凋零 以及火焰傷害) 所以這邊我改翻譯為 `防毒布料`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "item.thermal.nuke_grenade.desc": "記得扔遠點。",
+  "item.thermal.phyto_grenade": "植育一號催熟爆彈",
+  "item.thermal.phyto_grenade.desc": "可投擲的植育一號，作用範圍更廣。",
+  "item.thermal.redstone_grenade": "紅石發信彈",
```
這邊我不太肯定，但台灣這邊應該是使用 `發射` 或 `訊號發射`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.ender_tnt.keyword": "tnt",
+  "block.thermal.fire_tnt.keyword": "烈焰 火 tnt",
+  "block.thermal.glowstone_tnt.keyword": "tnt",
+  "block.thermal.ice_tnt.keyword": "暴雪人 冰 tnt",
```
這邊的 `人` 都沒有替換為 `使者`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "info.thermal.hazmat_helmet": "增加供氧量。",
+  "info.thermal.hazmat_boots": "減少摔落傷害。",
+
+  "item.thermal.beekeeper_fabric": "梭織布",
```
看 `Fabric` 要統一使用 `布` 還是 `布料`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.amaranth": "莧菜",
+  "block.thermal.amaranth_block": "莧菜捆",
+  "block.thermal.barley": "大麥",
+  "block.thermal.barley_block": "大麥垛",
```
都一樣是 `Bale`，統一譯作 `捆` 即可

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.eggplant": "茄子",
+  "block.thermal.eggplant_block": "箱裝茄子",
+  "block.thermal.flax": "亞麻",
+  "block.thermal.flax_block": "亞麻垛",
```
都一樣是 `Bale`，統一譯作 `捆` 即可

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.onion_block": "箱裝洋蔥",
+  "block.thermal.peanut": "花生",
+  "block.thermal.peanut_block": "袋裝花生",
+  "block.thermal.radish": "小蘿蔔",
```
從遊戲內圖片看起來，應該是 `櫻桃蘿蔔`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "item.thermal.onion_seeds": "洋蔥種子",
+  "item.thermal.peanut": "花生",
+  "item.thermal.peanut_seeds": "花生種子",
+  "item.thermal.radish": "小蘿蔔",
```
從遊戲內圖片看起來，應該是 `櫻桃蘿蔔`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.phytosoil.keyword": "作物 農場 蘑菇",
+  "block.thermal.phytosoil_tilled.keyword": "作物 農場",
+
+  "item.thermal.glowstone_mushroom.keyword": "螢石",
```
應為螢光石，只在粉型態時稱作 `螢石粉`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.energy_duct": "能量導管",
+  "block.thermal.fluid_duct": "流體導管",
+
+  "block.thermal.energy_distributor": "RF 分發界面",
```
`分配介面` 應該比較好

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+
+  "item.thermal.chiller_ball_cast": "球形鑄模",
+  "item.thermal.chiller_ingot_cast": "錠鑄模",
+  "item.thermal.chiller_rod_cast": "桿鑄模",
```
這邊統一使用 `O狀鑄模` 不曉得會不會比較好

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "item.thermal.press_packing_3x3_die": "打包模具（3x3）",
+  "item.thermal.press_unpacking_die": "解包模具",
+
+  "item.thermal.slot_seal": "欄位卡榫",
```
感覺可以不用太堅持 `Seal` 原意，能讓玩家清楚知道這是佔位的擋板就好了

Jimmy-sheep replied:
呃..那要直翻`佔位擋板`嗎?
我想說卡榫似乎也符合它的用途..

StarskyXIII replied:
窩噗基道，翻譯基本上都是看譯者當下的喜好為主 XD
如果你認為翻譯沒問題的話，那就保持原樣。

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+
+  "block.thermal.lumium_activator_rail": "流明觸發鐵軌",
+  "block.thermal.lumium_crossover_rail": "流明渡線鐵軌",
+  "block.thermal.lumium_detector_rail": "流明探測鐵軌",
```
`Detector Rail` 台灣這邊翻 `感測鐵軌`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.lumium_activator_rail": "流明觸發鐵軌",
+  "block.thermal.lumium_crossover_rail": "流明渡線鐵軌",
+  "block.thermal.lumium_detector_rail": "流明探測鐵軌",
+  "block.thermal.lumium_powered_rail": "流明充能鐵軌",
```
`Powered Rail` 台灣這邊翻 `動力鐵軌`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "block.thermal.prismarine_activator_rail": "海磷觸發鐵軌",
+  "block.thermal.prismarine_crossover_rail": "海磷渡線鐵軌",
+  "block.thermal.prismarine_detector_rail": "海磷探測鐵軌",
+  "block.thermal.prismarine_powered_rail": "海磷充能鐵軌",
```
`Powered Rail` 台灣這邊翻 `動力鐵軌`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+
+  "block.thermal.prismarine_activator_rail": "海磷觸發鐵軌",
+  "block.thermal.prismarine_crossover_rail": "海磷渡線鐵軌",
+  "block.thermal.prismarine_detector_rail": "海磷探測鐵軌",
```
`Detector Rail` 台灣這邊翻 `感測鐵軌`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "entity.thermal.lightning_tnt_minecart": "震天雷擊炸藥礦車",
+  "entity.thermal.nuke_tnt_minecart": "核彈礦車",
+  "entity.thermal.phyto_tnt_minecart": "PNT 礦車",
+  "entity.thermal.redstone_tnt_minecart": "紅石發信炸藥礦車",
```
這邊我不太肯定，但台灣這邊應該是使用 `發射` 或 `訊號發射`

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -0,0 +1,1330 @@
+  "item.thermal.lightning_tnt_minecart": "震天雷擊炸藥礦車",
+  "item.thermal.nuke_tnt_minecart": "核彈礦車",
+  "item.thermal.phyto_tnt_minecart": "PNT 礦車",
+  "item.thermal.redstone_tnt_minecart": "紅石發信炸藥礦車",
```
這邊我不太肯定，但台灣這邊應該是使用 `發射` 或 `訊號發射`

Commit:
Update zh_tw.json

According to the suggestion from StarskyXIII, fix some phraseology.

This translation, still need comfirm:
    "item.thermal.slot_seal": "欄為卡榫"

Thanks for StarskyXIII support the translations.
